app/src/test/java/com/codesoom/assignment/application/TaskServiceTest.java

```diff
@@ -0,0 +1,294 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.dto.TaskRequest;
+import com.codesoom.assignment.exceptions.BadRequestException;
+import com.codesoom.assignment.exceptions.NotFoundException;
+import org.junit.jupiter.api.*;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+@DisplayName("TaskServiceTest 클래스")
```

Comment:
```suggestion
@DisplayName("TaskServiceTest 클래스")
@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
```

이렇게 하고 아래있는 어노테이션은 전부 지워도 될 것 같아요!

Reply:
`@Nested`처럼 클래스마다 써줘야 하는 건 줄 알았는데 이렇게 해도 되는 거였군요!
불필요한 코드를 줄일 수 있어서 좋네요👍👍